src/ddoc_cache/src/ddoc_cache_entry_custom.erl

```diff
@@ -10,25 +10,28 @@
 % License for the specific language governing permissions and limitations under
 % the License.
 
--module(ddoc_cache_util).
+-module(ddoc_cache_entry_custom).
```

Comment:
Are you planing to remove it along with `_speed.erl` after you finish testing? 
If not then why not to dispatch the rest of the API functions to Mod?